### ISSUE TITLE
not bid/ask prices or volume for futures on binance

### ISSUE THREAD

**Issue by tomaszrydzewski** [ccxt/ccxt#9252]
OS: Win 10
Programming Language version: Python 3.7.4
CCXT version: 1.50.30

```


exchange_id = 'binance'
exchange_class = getattr(ccxt, exchange_id)
exchange = exchange_class({
                    'apiKey': '',
                    'secret': '',
                    'timeout': 30000,
                    'enableRateLimit': True,
                    'options': {
                        'defaultType': 'delivery',},
                })
exchange.load_markets()
data = exchange.fetch_tickers(['BTCUSD_210625'])
print(data)
```

```
{'BTCUSD_210625': {'symbol': 'BTCUSD_210625', 'timestamp': 1621932478954, 'datetime': '2021-05-25T08:47:58.954Z', 'high': 40135.9, 'low': 36068.8, 'bid': None, 'bidVolume': None, 'ask': None, 'askVolume': None, 'vwap': 38369.03262876, 'open': 36813.9, 'close': 38362.3, '
```

**Comment by frosty00** [comment]
yeah we'll fix this asap

**Comment by tomaszrydzewski** [comment]
Hi. Which version will this fix be release in ? I got the ccxt-1.50.37 from pip and still getting the same result.

**Comment by kroitor** [comment]
@tomaszrydzewski here's my output with 1.50.37:

```
~/ccxt $ python3 examples/py/cli.py binancecoinm fetch_ticker BTCUSD_210625
Python v3.8.2
CCXT v1.50.37
{'ask': None,
 'askVolume': None,
 'average': None,
 'baseVolume': 26528.528697,
 'bid': None,
 'bidVolume': None,
 'change': 2349.5,
 'close': 40149.5,
 'datetime': '2021-05-26T10:08:28.440Z',
 'high': 41169.2,
 'info': {'baseVolume': '26528.52869700',
          'closeTime': '1622023708440',
          'count': '315879',
          'firstId': '27000851',
          'highPrice': '41169.2',
          'lastId': '27316735',
          'lastPrice': '40149.5',
          'lastQty': '13',
          'lowPrice': '36624.6',
          'openPrice': '37800.0',
          'openTime': '1621937280000',
          'pair': 'BTCUSD',
          'priceChange': '2349.5',
          'priceChangePercent': '6.216',
          'symbol': 'BTCUSD_210625',
          'volume': '10232021',
          'weightedAvgPrice': '38569.87742089'},
 'last': 40149.5,
 'low': 36624.6,
 'open': 37800.0,
 'percentage': 6.216,
 'previousClose': None,
 'quoteVolume': 10232021.0,
 'symbol': 'BTCUSD_210625',
 'timestamp': 1622023708440,
 'vwap': 38569.87742089}
~/ccxt $
```

As you can see above, the `info` substructure contains the response from the exchange as is, and there is no bid/ask price in the ticker – you have to get that info from the orderbook with fetch_order_book. Some fields in the tickers may be missing, if the underlying exchange does not serve them, and this is documented in the Manual: https://github.com/ccxt/ccxt/wiki/Manual#notes-on-ticker-structure

Code to reproduce:

```Python
import ccxt
from pprint import pprint
print('CCXT Version:', ccxt.__version__)
exchange = ccxt.binancecoinm()
markets = await exchange.load_markets()
exchange.verbose = True
ticker = exchange.fetch_ticker('BTCUSD_210625')
pprint(ticker)
```

Let us know if that does not answer your question.
